### PR TITLE
Translation of 2021 Capabilities chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of [#586](https://github.com/HTTPArchive/almanac.httparchive.org/issues/586)

### CODE REVIEW

File: src/content/ja/2021/capabilities.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: ケイパビリティ
+description: 2021年版Web Almanacの「ケイパビリティ」の章では、Webアプリケーションにハードウェアインタフェースへのアクセスを提供し、Webベースの生産性アプリケーションを強化する、全く新しい強力なWebプラットフォームAPIをカバーしています。
+authors: [christianliebel]
+reviewers: [tomayac, hemanth]
+analysts: [tomayac]
+editors: [tunetheweb]
+translators: [ksakae1216]
+results: https://docs.google.com/spreadsheets/d/1b4moteB9EiLYkH1Ln9qfi1tnU-E4N2UQ87uayWytDKw/
+christianliebel_bio: Christian Liebel は、<a hreflang="en" href="https://thinktecture.com">Thinktecture</a> のコンサルタントで、さまざまなビジネス分野のクライアントが優れた Web アプリケーションを実装できるように支援しています。Microsoft MVP for Developer Technologies、Google GDE for Web/Capabilities and Angularを取得しており、W3C Web Applications Working Groupに参加しています。
```
Comment (tunetheweb) [inline src/content/ja/2021/capabilities.md:11]: Do the `for` and `and`s need translated?

File: src/content/ja/2021/capabilities.md [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+
+{{ figure_markup(
+image="fugu-podium.jpg",
+caption='フグのAPIをもっとも多く利用している3つのWebサイト。',
+description="画像は、フグのAPIをもっとも多く利用した3つのWebサイトのスクリーンショットが並べられた優勝壇上です。",
+width=1050,
+height=442
+) }}
+
+1. 1位は<a hreflang="en" href=<a hreflang="en" href="https://whatwebcando.today/">whatwebcando.today</a> で、28の機能を使用しています。HTML5のデバイス統合APIを、各機能のライブデモを提供することで、紹介しています。当然ながら、使用されているAPIの数は非常に多い。結果セットでは、<a hreflang="en" href="https://whatpwacando.today/">whatpwacando.today</a> という同様のサイトがPWA機能を紹介し、8つのAPIを使用していることがわかります。
```
Comment (tunetheweb) [inline src/content/ja/2021/capabilities.md:425]: That first link is wrong:

```suggestion
1. 1位は<a hreflang="en" href="https://whatwebcando.today/">whatwebcando.today</a> で、28の機能を使用しています。HTML5のデバイス統合APIを、各機能のライブデモを提供することで、紹介しています。当然ながら、使用されているAPIの数は非常に多い。結果セットでは、<a hreflang="en" href="https://whatpwacando.today/">whatpwacando.today</a> という同様のサイトがPWA機能を紹介し、8つのAPIを使用していることがわかります。
```

**Review by tunetheweb — CHANGES_REQUESTED** [review]
Looks good, apart from a couple of small comments.

File: src/content/ja/2021/capabilities.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2021/capabilities.md]: Thanks, I translated.

File: src/content/ja/2021/capabilities.md (hunk 2 above)
Comment (ksakae1216) [reply src/content/ja/2021/capabilities.md]: Thanks, I translated.

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**